Shell/Tests/brace-exp.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+setopt --verbose
+
+fail() {
+    echo $*
+    exit 1
+}
+
+test "$(echo {a,b,})" = "a b " || fail normal brace expansion with one empty slot
+test "$(echo {a,,b})" = "a  b" || fail normal brace expansion with one empty slot
+test "$(echo {a,,,b})" = "a   b" || fail normal brace expansion with two empty slots
+test "$(echo {a,b,,})" = "a b  " || fail normal brace expansion with two empty slots
+
+test "$(echo {a..c})" = "a b c" || fail range brace expansion, alpha
+test "$(echo {0..3})" = "0 1 2 3" || fail range brace expansion, number
+test "$(echo {س..ص})" = "س ش ص" || fail range brace expansion, unicode codepoint
```

Comment:
First instance of these codepoints in these tests, might be smarter to use codepoints that our fonts have glyphs for? I'm not entirely sure.

Reply:
How about emojis, like in this LibJS test testing Unicode capabilities:

https://github.com/SerenityOS/serenity/blob/4fb96afafc6e87b75d2e310f949b0ca7b337b050/Libraries/LibJS/Tests/string-escapes.js#L9-L15

Reply:
Do we have a consecutive range of emojis drawn?

Let's iterate some emojis instead!

Reply:
From emoji.txt

```
😂 - U+1F602 FACE WITH TEARS OF JOY
😃 - U+1F603 SMILING FACE WITH OPEN MOUTH
😄 - U+1F604 SMILING FACE WITH OPEN MOUTH AND SMILING EYES
```

Reply:
Even this test is happy about ranges of emojis :P

Reply:
😀

Reply:
😂 😃 😄